ros/src/computing/perception/detection/packages/cv_tracker/nodes/ssd/ssd_node.cpp

```diff
@@ -57,6 +57,9 @@ class RosSsdApp
 
 	//The minimum score required to filter the detected objects by the ConvNet
 	float score_threshold_;
+	std::string image_raw_topic_str;
+	std::string network_definition_file;
+	std::string pretrained_model_file;
```

Comment:
Please add a trailing underscor to member variables.
See http://wiki.ros.org/CppStyleGuide#Member_variables.